app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -69,6 +69,20 @@ class Config {
     return String.fromCharCodes(cacheValue).split(',');
   }
 
+  Future<List<Map<String, dynamic>>> _getLuciBuilders(String bucket) async {
```

Comment:
Do we also need the repo?

Reply:
Do we change that frequently? If this is needed, I would prefer to support that in a separate PR.

Reply:
sounds good to me to add in a different PR, the use case for this is that config files will now live in their own repositories. E.g engine configs will be pulled from the engine repo and sdk from flutter repo. This way those test will be branched out along with the repository code.

\cc @christopherfujino